apps/src/templates/courseOverview/resourceType.js

```diff
@@ -29,8 +28,3 @@ export const stringForType = {
   [ResourceType.videos]: i18n.videos(),
   [ResourceType.curriculumGuide]: i18n.curriculumGuide()
 };
-
```

Comment:
I think `stringForType` can also be removed but I'm not 100% sure

Reply:
good catch! will do in https://github.com/code-dot-org/code-dot-org/pull/47102